openedx/core/djangoapps/oauth_dispatch/management/commands/tests/test_clear_expired_tokens.py

```diff
@@ -99,8 +104,8 @@ def test_clear_expired_tokens(self):
         QuerySet.delete = counter(QuerySet.delete)
         try:
             call_command('edx_clear_expired_tokens', batch_size=1, sleep_time=0)
-            # three being the number of tables we'll end up unnecessarily calling .delete on once
-            assert QuerySet.delete.invocations == initial_count + 3  # pylint: disable=no-member
+            # four being the number of tables we'll end up unnecessarily calling .delete on once
```

Comment:
oof this is a little imprecise also; four is really the number of times we call `clear_table_data`, not the number of tables we're operating on here.

Reply:
I don't want to get too deep into the business of modifying the original code before seeing if we can just unblock it

Reply:
yeah seems fine, I shoulda' labelled this a nit